internal/payloads/error_payload.go

```diff
@@ -153,47 +151,47 @@ func NewClientError(ctx context.Context, err error) render.Renderer {
 	return NewResponseError(ctx, 500, "HTTP service returned unknown client error", err)
 }
 
-func NewNotFoundError(ctx context.Context, message string, err error) render.Renderer {
+func NewNotFoundError(ctx context.Context, message string, err error) *ResponseError {
 	message = fmt.Sprintf("Not found: %s", message)
 	return NewResponseError(ctx, http.StatusNotFound, message, err)
 }
 
-func NewEnqueueTaskError(ctx context.Context, message string, err error) render.Renderer {
+func NewEnqueueTaskError(ctx context.Context, message string, err error) *ResponseError {
 	message = fmt.Sprintf("Task enqueue error: %s", message)
 	return NewResponseError(ctx, http.StatusInternalServerError, message, err)
 }
 
-func NewDAOError(ctx context.Context, message string, err error) render.Renderer {
+func NewDAOError(ctx context.Context, message string, err error) *ResponseError {
 	message = fmt.Sprintf("DAO error: %s", message)
 	return NewResponseError(ctx, http.StatusInternalServerError, message, err)
 }
 
-func NewRenderError(ctx context.Context, message string, err error) render.Renderer {
+func NewRenderError(ctx context.Context, message string, err error) *ResponseError {
 	message = fmt.Sprintf("Rendering error: %s", message)
 	return NewResponseError(ctx, http.StatusInternalServerError, message, err)
 }
 
-func NewURLParsingError(ctx context.Context, message string, err error) render.Renderer {
+func NewURLParsingError(ctx context.Context, message string, err error) *ResponseError {
 	message = fmt.Sprintf("URL parsing error: %s", message)
 	return NewResponseError(ctx, http.StatusBadRequest, message, err)
 }
 
-func NewStatusError(ctx context.Context, message string, err error) render.Renderer {
+func NewStatusError(ctx context.Context, message string, err error) *ResponseError {
 	message = fmt.Sprintf("Status error: %s", message)
 	return NewResponseError(ctx, http.StatusInternalServerError, message, err)
 }
 
-func NewAWSError(ctx context.Context, message string, err error) render.Renderer {
+func NewAWSError(ctx context.Context, message string, err error) *ResponseError {
 	message = fmt.Sprintf("AWS API error: %s", message)
 	return NewResponseError(ctx, http.StatusInternalServerError, message, err)
 }
 
-func NewAzureError(ctx context.Context, message string, err error) render.Renderer {
+func NewAzureError(ctx context.Context, message string, err error) *ResponseError {
 	message = fmt.Sprintf("Azure API error: %s", message)
 	return NewResponseError(ctx, http.StatusInternalServerError, message, err)
 }
 
-func NewGCPError(ctx context.Context, message string, err error) render.Renderer {
+func NewGCPError(ctx context.Context, message string, err error) *ResponseError {
 	message = fmt.Sprintf("Google API error: %s", message)
 	return NewResponseError(ctx, http.StatusInternalServerError, message, err)
 }
```

Comment:
Oh I see that the `NewResponseError` refactoring was merged in other change, but the change of the return type to `Renderer` was a left-over (previously I planned to unexport the ReponseError type). This puts it back, I dont think we need to return `Renderer` interface at all.